x-pack/plugins/security_solution/public/management/components/administration_list_page.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React, { FC } from 'react';
+import { EuiPanel, EuiSpacer, CommonProps } from '@elastic/eui';
+import { SecurityPageName } from '../../../common/constants';
+import { WrapperPage } from '../../common/components/wrapper_page';
+import { HeaderPage } from '../../common/components/header_page';
+import { SiemNavigation } from '../../common/components/navigation';
+import { SpyRoute } from '../../common/utils/route/spy_routes';
+import { AdministrationSubTab } from '../types';
+import { ENDPOINTS_TAB, TRUSTED_APPS_TAB, BETA_BADGE_LABEL } from '../common/translations';
+import { getEndpointListPath, getTrustedAppsListPath } from '../common/routing';
+
+interface AdministrationListPageProps {
+  beta: boolean;
+  title: React.ReactNode;
+  subtitle: React.ReactNode;
+  actions?: React.ReactNode;
+}
+
+export const AdministrationListPage: FC<AdministrationListPageProps & CommonProps> = ({
```

Comment:
Suggestion: Use `memo()` in defining your functional component in order to help with re-renders (or avoiding them if no prop changes)

Reply:
The way I interpret official [documentation](https://reactjs.org/docs/hooks-reference.html#usememo) is that it should be used when optimising performance problem, why do we use it everywhere? For me it reduces readability pretty significantly. Also I'd rather use it on return value only than on the whole component, that at least keeps readability of component definition better.

Reply:
I agree with using `memo()` where possible.  Kibana is a big application with sometimes unforeseen side effects that cause re-renders.  You could add it in in the next PR though.

Reply:
using it does not hurt, but not using it implies that everyone is aware of the performance "gotchas" that come with React development. IMO - the safer way is to memoize it.

Reply:
added